core/trino-main/src/main/java/io/trino/testing/MaterializedResult.java

```diff
@@ -101,7 +101,12 @@
 
     public MaterializedResult(List<MaterializedRow> rows, List<? extends Type> types)
     {
-        this(rows, types, ImmutableList.of(), ImmutableMap.of(), ImmutableSet.of(), Optional.empty(), OptionalLong.empty(), ImmutableList.of(), Optional.empty());
+        this(rows, types, ImmutableList.of());
```

Comment:
empty list is not a valid value for column names, let's use `Optional<List>`

Reply:
BTW this old constructor is used on one place only, let's remove it.

Reply:
> BTW this old constructor is used on one place only, let's remove it.

Old constructor with 2 arguments is being used multiple (5) times.